scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -109,23 +109,15 @@ RUN set -ex; \
 	ninja install/strip; \
 	rm -rf /usr/src/hera
 
-# ANTLR4 RUNTIME
+# libabicoder
 RUN set -ex; \
     cd /usr/src; \
-    wget https://www.antlr.org/download/antlr4-cpp-runtime-4.8-source.zip; \
-    rm -rf antlr4-runtime && mkdir antlr4-runtime; \
-    unzip antlr4-cpp-runtime-4.8-source.zip -d antlr4-runtime; \
-    cd antlr4-runtime && mkdir build && cd build; \
-    cmake .. -DWITH_LIBCXX=On -DCMAKE_BUILD_TYPE=Release -DWITH_DEMO=False; \
-    make -j; \
-    DESTDIR=run make install; \
-    # Manually copy needed library and includes since install script
-    # does not respect -DCMAKE_INSTALL_DIR and there is no option
-    # to disable shared library build/installation
-    cp -Rf run/usr/local/include/antlr4-runtime /usr/include; \
-    cp -f run/usr/local/lib/libantlr4-runtime.a /usr/lib; \
-    rm -rf /usr/src/antlr4-cpp-runtime-4.8-source.zip; \
-    rm -rf /usr/src/antlr4-runtime
+    git clone https://github.com/ekpyron/Yul-Isabelle; \
+    cd Yul-Isabelle; \
+    cd libabicoder; \
+    CXX=clang++ CXXFLAGS="-stdlib=libc++" make; \
```

Comment:
Since that's how boost is build I would guess this is what we'll need in the end.

Reply:
I'm actually not removing ``/usr/src/Yul-Isabelle/`` afterwards here - but I think that's fine, since these are temporary layers anyways and below it only specifically copies ``/usr/{lib,bin,include}`` anyways.

Reply:
I'll confirm once my local build went through entirely - just wanted to push this early, s.t. CI already starts building :-).

Reply:
Actually I added the removal - maybe cleaner either way.